rest_framework_social_oauth2/management/commands/createapp.py

```diff
@@ -0,0 +1,33 @@
+from django.core.management.base import BaseCommand, CommandError
+from oauth2_provider.models import Application
+from django.contrib.auth.models import User
+from oauth2_provider.generators import generate_client_id, generate_client_secret
+
+
+class Command(BaseCommand):
+    help = "Create a Django OAuth Toolkit application (an existing admin is required)"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-ci", "--client_id",
+            help="Client ID (recommeded 40 characters long)"
+        )
+        parser.add_argument(
+            "-cs", "--client_secret",
+            help="Client Secret (recommeded 128 characters long)"
+        )
+        parser.add_argument(
+            "-n", "--name",
+            help="Name for the application"
+        )
+
+    def handle(self, *args, **options):
+        new_application = Application(
+            user= User.objects.filter(is_superuser=True)[0],
+            client_type="confidential",
```

Comment:
Shouldn't these fields (`client_type`, `authorization_grant_type`) be also set as arguments in CLI? Perhaps we can default to `password` and `socialauth_application` if user does not provide them.